hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1864,4 +1864,14 @@ possible configurations would overwhelm and obscure the important.
     <description>Default is 5 minutes. Make it 30 seconds for tests. See
     HBASE-19794 for some context.</description>
   </property>
+  <property>
+    <name>hbase.master.cleaner.snapshot.interval</name>
+    <value>600000</value>
```

Comment:
nit:
Not a strong opinion, I think 10m is too frequent for snapshot cleanup chore.

Reply:
Should we keep 30 min or 1 hr?